ddl/reorg.go

```diff
@@ -134,6 +136,44 @@ func (rc *reorgCtx) getRowCount() int64 {
 	return row
 }
 
+func getDistAddIndexRowCntAndSetJobRowCnt(ctx context.Context, reorgInfo *reorgInfo, rc *reorgCtx, job *model.Job, client *clientv3.Client) int64 {
```

Comment:
Renaming it into `setJobRowCount` is better, because not only distributed row count is updated.